R/parameters.R

```diff
@@ -0,0 +1,208 @@
+#' Read in disease process parameters from an external file or files
+#'
+#' @param generation_interval_path,delay_interval_path,right_truncation_path
+#'   Path to a local file with the parameter PMF. See [read_interval_pmf] for
+#'   details on the file schema. The parameters can be in the same file or a
+#'   different file.
+#' @param disease One of `COVID-19` or `Influenza`
+#' @param as_of_date The date "as of" the run. Set for the current date for the
+#'   most up-to-to date version of the parameters and set to an earlier date to
+#'   use parameters from an earlier time period.
+#' @param state Used only for right-truncation, the two-letter uppercase state
```

Comment:
Why only used for right truncation?

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/16/commits/ac9b445a81a289e1ae11e2400f37609c01ce451d